pkg/api/api.go

```diff
@@ -113,12 +115,31 @@ func NewAPI() (*API, error) {
 		return nil, errors.Wrap(err, "new verifier")
 	}
 
+	// verify cert chain if present
+	if len(certChain) > 0 {
+		roots := x509.NewCertPool()
+		intermediates := x509.NewCertPool()
+		for _, cert := range certChain[1:(len(certChain) - 1)] {
```

Comment:
Would this easier overall if the API served the cert and chain in separate fields? I can't decide myself but might have done that in Fulcio.

Reply:
Yes!! WDYT about the spinning up the rekor server -- should you provide a timestamp cert and a root CA cert separately too? since i guess the root CA cert can also be used in other cert chains